src/esrs/aggregate.rs

```diff
@@ -113,3 +134,73 @@ pub trait Aggregate: Identifier {
             })?)
     }
 }
+
+/// The Aggregate trait is responsible for validating commands, mapping commands to events, and
+/// mapping commands to events.
```

Comment:
...and mapping commands to events :smile:

Reply:
the `and` is on the previous line 😀